src/components/users/UsersIndex.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component, useReducer } from 'react';
+import Table from '../Table';
+
+// library for AJAX functions
+const axios = require('axios');
+
+// This component is a class because it has state. Having state is neccesary
+// because the page will render before the AJAX is complete. Once the AJAX
+// is processed and returned from backend we setState() which will automatically
+// rerender the component with the new state. An additional reason this
+// componenet has state is because if AJAX fails, no results will be available
+// to render
+class UsersIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    // initial state which will get updated from AJAX
+    this.state = {
+      index: []
+    };
+    
+    this.getUsers = this.getUsers.bind(this);
+
+  }
+  
+  // after component mounts ( mount is React term for after the initital render)
+  componentDidMount() {
+
+    // call function to send AJAX to backend
+    this.getUsers();
+  }
+
+  getUsers() {
```

Comment:
If you use an arrow function:

`getUsers = () => {}`

You don't need to call `.bind` as arrow functions preserve context

Reply:
When I change 29 to "getUsers = () => {}" , I get the error:  'getUsers' is not definded. Or should I do something else?

Reply:
Ah, we're missing `babel-eslint` as our parser - feel free to leave as is and we can change it in a follow up

Reply:
`babel` is what JavaScript uses to transform things like React syntax and modern JS to older ones, so we need to add the parser to `aslant`